content/en/blog/2024/k8s-book-club/k8s-book-club.md

```diff
@@ -0,0 +1,124 @@
+------
+layout: blog
+title: "A look into the Kubernetes Book Club"
+slug: k8s-book-club
+date: 2024-02-22
+author: "Frederico Muñoz (SAS Institute)"
+---
+
+Learning Kubernetes and the entire ecosystem of technologies around it is not without its
+challenges. In this interview, we will talk with [Carlos Santana
+(AWS)](https://www.linkedin.com/in/csantanapr/) to learn a bit more about how he created the
+[Kubernetes Book Club](https://community.cncf.io/kubernetes-virtual-book-club/), how it works, and
+how anyone can join in to take advantage of a community-based learning experience.
+
+![Carlos Santana speaking at KubeCon NA 2023](csantana_k8s_book_club.jpg)
```

Comment:
This image isn't rendering - please check that.

(also, I still wondered if this is actually Photoshopped - mostly an aside though)

Reply:
That was because I accepted a simplification change in `website` that makes sense only there. 

@csantanapr , could you elucidate @sftim on the nature of the provided image?

Reply:
@fsmunoz @sftim Here is the original one, I was trying to be funny, we could switched it to the original one if there is a problem with the photoshop, or add the text to the image alt of original source of image

https://www.flickr.com/photos/143247548@N03/53322630907/in/album-72177720312486917/
![Carlos at Kubecon](https://live.staticflickr.com/65535/53322630907_798eb2c7e1_k.jpg)

Reply:
@sftim , I'm unsure if it's implied that we should change the photo or if it's OK to keep it as is. If I'm not mistaken, this is the only pending topic.

Reply:
I'd prefer not to have the Photoshopped one on our blog, so please consider switching it (on both sites). That can happen before or after a merge.

Here's why: it's hard to tell if it's a dodgy Photoshop, intended as irony, or the actual photo. And I think we should be clear which one we want.

(for a non-dodgy photoshop, match the existing background gradient and use a perspective tool to match the existing skew)

---

Another option: don't mirror the article. If the audience is the contributor blog only, that image is OK as-is.

Reply:
Understood! Done, I've updated the photo on both PRs.